15/slim/debian/Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM debian:buster-slim
+
+ARG version=15.0.0.36-1
+# In addition to installing the Amazon corretto, we also install
+# fontconfig. The folks who manage the docker hub's
+# official image library have found that font management
+# is a common usecase, and painpoint, and have
+# recommended that Java images include font support.
+#
+# See:
+#  https://github.com/docker-library/official-images/blob/master/test/tests/java-uimanager-font/container.java
+#
+# Slim:
+#   JLink is used (retaining all modules) to create a slimmer version of the JDK excluding man-pages, header files and debugging symbols - saving ~113MB.
+RUN set -eux \
+    && apt-get update \
+    && apt-get install -y --no-install-recommends \
+        curl ca-certificates gnupg software-properties-common fontconfig java-common \
+    && curl -fL https://apt.corretto.aws/corretto.key | apt-key add - \
+    && add-apt-repository 'deb https://apt.corretto.aws stable main' \
+    && mkdir -p /usr/share/man/man1 || true \
+    && apt-get update \
+    && apt-get install -y java-15-amazon-corretto-jdk=1:$version \
+    && apt-get install -y binutils-i586-linux-gnu \
```

Comment:
Why `binutils-i586-linux-gnu` and not `binutils-i686-linux-gnu` or just `binutils`? Also why isn't this grouped into the same invocation with the line above it?